zkevm-circuits/src/tx_circuit.rs

```diff
@@ -2367,7 +2600,30 @@ impl<F: Field> TxCircuitConfig<F> {
         let (access_list_address_size, access_list_storage_key_size) =
             access_list_size(&tx.access_list);
 
-        // fixed_rows of a tx
+        // Only bytes from L2 txs are accumulated for chunk bytes hash
+        let is_chunk_bytes = (tx.tx_type != TxType::L1Msg) && !tx.caller_address.is_zero();
+
+        let hash_len = if is_chunk_bytes {
+            tx.rlp_signed.len()
+        } else {
+            0
+        };
+        let tx_hash_rlc = rlc_be_bytes(&tx.rlp_signed, keccak_input);
+        if hash_len >= pows_of_rand.len() {
+            for _ in 0..(tx.rlp_signed.len() - pows_of_rand.len() + 1) {
+                pows_of_rand.push(pows_of_rand.last().unwrap().mul(keccak_input));
```

Comment:
`pows_of_rand.push(pows_of_rand.last().unwrap().mul(keccak_input));` @kunxian-xia  we need to double check this line.   It is usually a dangerous pattern to unwrap challenge Value